internal/services/hybridkubernetes/hybrid_kubernetes_connected_cluster_resource.go

```diff
@@ -0,0 +1,206 @@
+package hybridkubernetes
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/hashicorp/go-azure-helpers/lang/response"
+	"github.com/hashicorp/go-azure-helpers/resourcemanager/commonschema"
+	"github.com/hashicorp/go-azure-helpers/resourcemanager/identity"
+	tagsHelper "github.com/hashicorp/go-azure-helpers/resourcemanager/tags"
+	"github.com/hashicorp/terraform-provider-azurerm/helpers/azure"
+	"github.com/hashicorp/terraform-provider-azurerm/helpers/tf"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/clients"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/location"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/services/hybridkubernetes/sdk/2021-10-01/hybridkubernetes"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/tags"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/tf/pluginsdk"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/timeouts"
+	"github.com/hashicorp/terraform-provider-azurerm/utils"
+)
+
+func resourceHybridKubernetesConnectedCluster() *pluginsdk.Resource {
+	return &pluginsdk.Resource{
+		Create: resourceHybridKubernetesConnectedClusterCreate,
+		Read:   resourceHybridKubernetesConnectedClusterRead,
+		Update: resourceHybridKubernetesConnectedClusterUpdate,
+		Delete: resourceHybridKubernetesConnectedClusterDelete,
+
+		Timeouts: &pluginsdk.ResourceTimeout{
+			Create: pluginsdk.DefaultTimeout(30 * time.Minute),
+			Read:   pluginsdk.DefaultTimeout(5 * time.Minute),
+			Update: pluginsdk.DefaultTimeout(30 * time.Minute),
+			Delete: pluginsdk.DefaultTimeout(30 * time.Minute),
+		},
```

Comment:
are these timeouts valid for this service?

Reply:
Yes, I have already test this resource that each operation will take no more than 10 seconds in normal situation. So I think these values are valid.